reana/reana_dev/git.py

```diff
@@ -724,95 +726,92 @@ def git_push(component):  # noqa: D301
             run_command(cmd, component)
 
 
-@git_commands.command(name="git-shared-modules-upgrade")
+@git_commands.command(name="git-upgrade-shared-modules")
 @click.option(
-    "--commit-and-publish",
+    "--component",
+    "-c",
+    multiple=True,
+    default=["CLUSTER", "CLIENT"],
+    help="Which components? [shortname|name|.|CLUSTER|ALL]",
+)
+@click.option(
+    "--use-latest-known-tag",
+    is_flag=True,
+    default=False,
+    help="Should the upgrade use the latest known tag of the shared modules?"
+    " If so, the latest known tag of the shared modules will be used to upgrade the"
+    " provided components. Else, the upgrade will only occur if the latest commit of the"
+    " shared modules points at a tag, in other case, the command will be aborted.",
+)
+@click.option(
+    "--create-new-version-bump-commit",
```

Comment:
Upon second thought, what about inverting the logic and creating new version bump commit by default unless `--amend` is passed? Advantage: it would behave just like `git`, so less surprises for people.

If we like it, then:

- let's rename `--create-new-version-bump-commit` to `--amend`
- let's make it false by default
- let's invert the implementation to always make release commits unless `--amend` is used.

WDYT?